interface/modules/custom_modules/oe-module-claimrev-connect/src/ClaimRevModuleSetup.php

```diff
@@ -0,0 +1,96 @@
+<?php
+    namespace OpenEMR\Modules\ClaimRevConnector;
+    require_once "../../../../globals.php";
```

Comment:
remove this require_once
(globals.php is always called in scripts that use libraries and classes, so do not need to redo it)